### PR TITLE
Put water component first.

### PR DESCRIPTION
**Pull request by atgeirr**
This will put the water equation first in Flow. Based on #325, replacing #324.

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by andlaus** [comment]
In principle I'm okay to merge this once the the bug mentioned in OPM/opm-simulators#1754 has been fixed, i.e., `ebos` and `ebos_altidx` produce identical results (modulo numerical noise) . I'd still advice you to implement an `ebos_cpr` variant for this instead of dragging everybody along, but I'm not dogmatic about this issue as long as somebody else is willing to take the blame in case something blows up...

File: opm/material/fluidsystems/BlackOilDefaultIndexTraits.hpp [hunk 1]
```diff
@@ -37,16 +37,16 @@ class BlackOilDefaultIndexTraits
 {
 public:
     //! Index of the water phase
-    static const unsigned waterPhaseIdx = 0;
+    static const unsigned oilPhaseIdx = 0;
     //! Index of the oil phase
```
Comment (andlaus) [inline opm/material/fluidsystems/BlackOilDefaultIndexTraits.hpp]: the comments need to be adapted as well

**Comment by andlaus** [comment]
I guess that with OPM/opm-simulators#1757 merged, this is no longer required. Closing tentatively.

**Pull request closed without merging**